### PR TITLE
Remove ratelimit

### PR DESCRIPTION
**Pull request by upacyxou**
- remove ratelimit
- add get route

### CODE REVIEW

File: src/helpers/proccessBlocksForNotifications.ts [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+import CID from '@/models/CID'
+import generateRandomName from '@/helpers/generateRandomName'
+import parsePostLogData from '@/helpers/parsePostLogData'
+import structToCid from '@/helpers/structToCid'
+
+export default function (posts: FeedPostAddedEvent[]) {
+  return posts
+    .map(({ data, topics }) => parsePostLogData({ data, topics }))
+    .map(({ args }) => args as [BigNumber, BigNumber, [string, CID, BigNumber]])
+    .map(([idFeed, postId, [author, metadata, commentFeedId]]) => {
```
Comment (T-Damer) [inline src/helpers/proccessBlocksForNotifications.ts:12]: Can we do only one `.map` instead or three?

File: src/helpers/runNotificationListener.ts [hunk 2]
```diff
@@ -4,41 +4,27 @@ import sendAppleNotification from '@/helpers/sendAppleNotification'
...
 const rootFeeds = {
   1: 't/startups',
   2: 't/ketlTeam',
 } as { [key: number]: string }
 
 obssContract.on('FeedPostAdded', async (feedId) => {
-  const currentTime = new Date()
   const title = rootFeeds[feedId.toNumber()]
     ? `Someone posted at ${rootFeeds[feedId.toNumber()]}`
     : undefined
```
Comment (T-Damer) [inline src/helpers/runNotificationListener.ts:16]: Do we need `undefined` title here

**Review by T-Damer — APPROVED** [review]

File: src/helpers/runNotificationListener.ts (hunk 2 above)
Comment (upacyxou) [reply src/helpers/runNotificationListener.ts:16]: yes

File: src/helpers/proccessBlocksForNotifications.ts (hunk 1 above)
Comment (smyzin) [inline src/helpers/proccessBlocksForNotifications.ts:12]: Can it be simplified?

**Review by smyzin — APPROVED** [review]

**Pull request merged**